__tests__/HomeTest.js

```diff
@@ -4,5 +4,5 @@ import Home from "../pages/index";
 
 it("Renders appropriately", () => {
   render(<Home />);
-  expect(screen.getByRole("heading", { name: "Babitz" })).toBeInTheDocument();
+  expect(screen.getByText( "Babitz" )).toBeInTheDocument();
```

Comment:
Edit test case, as code for index.js changed

Reply:
Can we also add a check for the login button here